src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -51,28 +45,35 @@ def _complex_params(command_group, engine):
             c.argument('value', help='Value of the configuration. If not provided, configuration value will be set to default.', validator=configuration_value_validator)
             c.ignore('source')
 
+        with self.argument_context('{} server wait'.format(command_group)) as c:
+            c.ignore('created', 'deleted', 'updated')
+
     _complex_params('mysql', mysql)
     _complex_params('postgres', postgresql)
 
     for scope in ['mysql', 'postgres']:
         with self.argument_context(scope) as c:
-            c.argument('name', options_list=['--sku-name'])
+            c.argument('name', options_list=['--sku-name'], required=True)
             c.argument('server_name', completer=server_completers[scope], options_list=['--server-name', '-s'], help='Name of the server.')
 
     for scope in ['mysql server', 'postgres server']:
         with self.argument_context(scope) as c:
+            c.ignore('size', 'family', 'capacity', 'tier')
+
             c.argument('server_name', options_list=['--name', '-n'], id_part='name', help='Name of the server.')
             c.argument('administrator_login', options_list=['--admin-user', '-u'])
             c.argument('administrator_login_password', options_list=['--admin-password', '-p'], required=False, help='The password of the administrator login.')
             c.argument('ssl_enforcement', arg_type=get_enum_type(['Enabled', 'Disabled']), options_list=['--ssl-enforcement'], help='Enable ssl enforcement or not when connect to server.')
-            c.argument('tier', arg_type=get_enum_type(['Basic', 'Standard']), options_list=['--performance-tier'], help='The performance tier of the server.')
-            c.argument('capacity', options_list=['--compute-units'], type=int, help='Number of compute units.')
+            c.argument('tier', arg_type=get_enum_type(['Basic', 'GeneralPurpose', 'MemoryOptimized']), required=False, options_list=['--performance-tier'], help='The performance tier of the server.')
```

Comment:
Is there no enum in your SDK for this?  Also, why do you have the kwarg `required=False` so many times?  In the overwhelming majority of cases, it shouldn't be needed.

Reply:
Done